spec/osut_tests_spec.rb

```diff
@@ -2909,8 +2915,9 @@ module M
     expect(core.floorArea).to be_within(TOL).of(149.66 - 4)      # -mini m2
     core_volume = core.floorArea * 3.05
     expect(core_volume).to be_within(TOL).of(core.volume)
-    expect(attic.volume).to be_within(TOL).of(720.19 + 4 * 3.05) # +mini m3
-    expect(attic.floorArea).to be_within(TOL).of(567.98)
+    expect(attic.volume).to be_within(TOL).of(720.19 + 4 * 3.05) unless v < 350
+    expect(attic.volume).to be_within(TOL).of(798.41 + 4 * 3.05)     if v < 350
+    expect(attic.floorArea).to be_within(TOL).of(567.98) # includes overhangs
```

Comment:
Adding an attic _well_ is correctly reflected in both area and volume calculations for both core and attic spaces.

The _US DOE SmallOffice Prototype_ attic space has indeed an initial volume of 720.19 m3. Not sure why SDK versions prior to 3.5.0 are off (a deeper dive into committed fixes seems warranted). These pre-v351 calculations errors are noted in previous tests.

Reply:
This PR https://github.com/NREL/OpenStudio/pull/4592 updated the OS volume calculations to match E+.  Previously the OS volume calculations were not accurate for more complex zone geometry.

Reply:
That is awesome. Thanks for the link, @macumber !